src/pages/tokenDrop/Landing.tsx

```diff
@@ -0,0 +1,203 @@
+  return (
+    <Box display="flex" flexDirection="column">
+      <Box mb={ 4 }>
+        <Typography variant="subtitle1">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis sed nibh
+          sit amet mi euismod pulvinar. Maecenas pulvinar lorem vel erat
+          efficitur, interdum ultrices magna ullamcorper. Nam imperdiet nibh
+          semper eros iaculis dictum. Donec non sapien sit amet tortor tincidunt
+          varius. Etiam hendrerit, felis eleifend maximus ultricies, ligula eros
+          maximus enim, non congue quam nisl id turpis. Ut eget fermentum massa.
+          Proin fermentum porttitor ipsum sit amet interdum. Vestibulum lacinia
+          sagittis malesuada. Fusce eget feugiat sapien. Proin eu sem vitae
+          tortor sagittis ornare. Quisque tempus libero id accumsan sodales.
+          Vivamus quam mi, molestie a lobortis maximus, bibendum nec nunc.
+        </Typography>
+      </Box>
+
+      <Stack
+        spacing={ 2.5 }
+        alignItems="flex-start"
+        width="100%"
+        maxWidth={ [9999, 9999, 450] }
+      >
+        <Box width="100%">
+          <SectionHeading>AVAILABLE SONG</SectionHeading>
+
+          <Box
+            sx={ {
+              display: "flex",
+              flexDirection: "row",
+              alignItems: "center",
+              justifyContent: "space-between",
+              width: "100%",
+              backgroundColor: theme.colors.grey600,
+              mt: 1,
+              pr: 2,
+            } }
+          >
+            <Stack spacing={ 2 } direction="row" alignItems="center">
+              <Box
+                display="flex"
+                height={ 2 }
+                width={ 2 }
+                margin={ 3 }
+                justifyContent="center"
+                alignItems="center"
+              >
+                { isPlaying ? (
+                  <IconButton
+                    aria-label="pause MURS song"
+                    onClick={ handleStopSong }
+                  >
+                    <StopIcon />
+                  </IconButton>
+                ) : (
+                  <IconButton
+                    aria-label="play MURS song"
+                    onClick={ handlePlaySong }
+                  >
+                    <PlayIcon />
+                  </IconButton>
+                ) }
+              </Box>
+
+              <img
+                src={ mursProfileImageXs }
+                style={ { width: 40, height: 40, borderRadius: "50%" } }
+                alt="murs profile"
+              />
+
+              <Box>
+                <Typography variant="h4" fontWeight={ 700 }>
+                  Break up
+                </Typography>
+                <Typography variant="subtitle2">MURS</Typography>
+              </Box>
+            </Stack>
+
+            <a
+              href="https://open.spotify.com/track/1ZSwTbIdB0p7pNLDck5RQb?si=b1c674b4020d4e90"
+              target="_blank"
+              rel="noreferrer"
```

Comment:
**Action Requested:**
I believe starting 2021 modern browser versions now protect against a major vulnerability and add noopener by default, but i am wondering if we should add that to our links to protect users in older browsers? so it would be: `rel="noopener noreferrer"`